pages.ja/common/docker.md

```diff
@@ -0,0 +1,37 @@
+# docker
+
+> Dockerコンテナ及びDockerイメージの管理を行います。
+> `docker run`などいくつかのサブコマンドがあります。これらには使用方法についての独自のドキュメントがあります。
```

Comment:
We have a [template](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/subcommand-mention.md#ja) for this.
```suggestion
> `docker run` のようないくつかのサブコマンドには、使用方法についての独自のドキュメントがあります。
```